sym/provider/flows_filter_resource.go

```diff
@@ -0,0 +1,142 @@
+package provider
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/symopsio/terraform-provider-sym/sym/client"
+	"github.com/symopsio/terraform-provider-sym/sym/utils"
+)
+
+// FlowsFilter Resource
+//
+// This resource allows customers to specify which flows can be shown to which users
+// using a Python `implementation` file
+func FlowsFilter() *schema.Resource {
+	return &schema.Resource{
+		Description:   "The `sym_flows_filter` resource provides a FlowsFilter that can be used to filter out flows for certain users. You may only have one FlowsFilter resource in your organization.",
```

Comment:
```suggestion
		Description:   "The `sym_flows_filter` resource provides a FlowsFilter that can be used to filter the Flows displayed to the requester. You may only have one FlowsFilter resource in your organization.",
```

Capitalize Flows, and wondering if we should specify that it's filtering for the requester?